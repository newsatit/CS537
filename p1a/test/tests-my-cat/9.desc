many files on command line with one of them being empty file
