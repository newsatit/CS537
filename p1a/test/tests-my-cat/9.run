./my-cat INPUT_DIR/9a.in INPUT_DIR/9b.in INPUT_DIR/9c.in INPUT_DIR/9d.in
