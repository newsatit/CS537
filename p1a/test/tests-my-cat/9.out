this one has stuff in it
this one does too
this one should be printed without preceding empty line
